dataflow/src/domain/mod.rs

```diff
@@ -1446,15 +1450,16 @@ impl Domain {
         }
 
         let n = self.nodes[&source].borrow();
+        let data: &Vec<DataType> = &*row;
```

Comment:
This seems like an unnecessary change?

Reply:
It ends up cloning the `Row` without it, probably because `Row` now derives `Clone` as well (which is needed by `Cow`). I changed it to double deref `row` further down instead of the variable though, seemed cleaner. 

https://github.com/mit-pdos/distributary/pull/72/files#diff-f3207323d6f34ad3c124fbcb4cb87a40R1477